doc/source/release-notes/Percona-Server-8.0.15-5.rst

```diff
@@ -0,0 +1,86 @@
+.. rn:: 8.0.15-5
+
+================================================================================
+|Percona Server| |release|
+================================================================================
+
+|Percona| announces the release of |Percona Server| |release| on |date|
+(downloads are available `here
+<https://www.percona.com/downloads/Percona-Server-8.0/>`__ and from the `Percona
+Software Repositories
+<https://www.percona.com/doc/percona-server/8.0/installation.html#installing-from-binaries>`__).
+
+This release includes fixes to bugs found in previous releases of |Percona
+Server| 8.0. In previous releases, the audit log used to produce time stamps
```

Comment:
Consider linking to feature docs

Is this prominent enough as it goes to the historically "blah blah" section of the release notes? Does it need an own section of "incompatible changes?"

Reply:
Ready for review: added a note like section titled "Incompatible changes"